sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/TooManyParameters.cs

```diff
@@ -84,6 +84,14 @@ private static bool CanBeChanged(SyntaxNode node, SemanticModel semanticModel)
                 return true;
             }
 
+            if ((node as ConstructorDeclarationSyntax)?.Initializer?.ArgumentList?.Arguments.Count > Maximum)
+            {
+                // Base class is already not compliant so let's ignore current constructor.
+                // Another option could be to substract current number of parameters from base count and raise only if greater
+                // than threshold.
+                return false;
+            }

```

Comment:
For consistency, should we not raise the issue on overridden methods too?

Reply:
It is already handled (interface implementation + override) on the next lines (through semantic model).